### PR TITLE
Update readme and helm chart.yaml

### PR DESCRIPTION
**Pull request by muvaf**
### Description of your changes

UI of https://github.com/upbound/universal-crossplane/pull/96 is broken for some reason.

Fixes #

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,39 @@ production control planes.
...
+
+1. Install the latest stable version of UXP.
+
+   ```console
+   helm install uxp --namespace upbound-system upbound-stable/universal-crossplane --devel
+   ```
+
+#### Upgrade from upstream Crossplane
+
+1. Upgrade the existing installation of Crossplane with UXP chart.
```
Comment (hasheddan) [inline README.md:71]: Should we also add an Upgrade with `up` example and mention that versions must match?

**Comment by muvaf** [comment]
@hasheddan ready for another look!

File: README.md [hunk 2]
```diff
@@ -42,6 +42,49 @@ production control planes.
...
+
+   ```console
+   helm install uxp --namespace upbound-system upbound-stable/universal-crossplane --devel
+   ```
+
+### Upgrade from upstream Crossplane
+
+#### Using up CLI
+
+1. Upgrade the Crossplane installation in given namespace.
```
Comment (hasheddan) [inline README.md:73]: nitpick: this sounds a little bit like you are just upgrading to a new Crossplane version. I wonder if we could just remove the text from these sections given that there is only one step?

**Review by hasheddan — APPROVED** [review]
One nitpick, but feel free to address as you see fit :)

File: README.md (hunk 2 above)
Comment (muvaf) [reply README.md]: Removed and added some details about upgrade and version match.

**Review by hasheddan — APPROVED** [review]

**Pull request merged**